### PR TITLE
Articles on Tensor Networks and part of the aQa glossary

### PR DESCRIPTION
**Pull request by patrickemonts**
Notes on MPS, PEPS and some more references for the TN

### CODE REVIEW

File: vault/Ansatz.md
Comment (AdrianPerezSalinas) [inline vault/Ansatz.md:1]: I would change "course" for "context" or something more general

File: vault/QAOA.md
Comment (AdrianPerezSalinas) [inline vault/QAOA.md:1]: Maybe we should add a link to VQE, another foundational paper

File: vault/Ground State Problem.md
Comment (AdrianPerezSalinas) [inline vault/Ground State Problem.md:1]: Let us add a note with just "Ground state", (and maybe another one with excited state, that we can connect to other works)

File: vault/Variational Quantum Algorithm (VQA).md
Comment (AdrianPerezSalinas) [inline vault/Variational Quantum Algorithm (VQA).md:1]: Add these references 

https://journals.aps.org/rmp/abstract/10.1103/RevModPhys.94.015004
https://www.nature.com/articles/s42254-021-00348-9

Comment (AdrianPerezSalinas) [reply vault/Variational Quantum Algorithm (VQA).md:1]: (might be a little bit biased)

**Review by AdrianPerezSalinas — COMMENTED** [review]
A couple of small changes, and then we can merge

File: vault/Ansatz.md
Comment (patrickemonts) [reply vault/Ansatz.md:1]: That was a copy-paste problem. Fixed in https://github.com/patrickemonts/zettelkasten/pull/4/commits/c49ff41fe18dcfe1479db1cac2419a29858e132c

File: vault/QAOA.md
Comment (patrickemonts) [reply vault/QAOA.md:1]: Added link to VQE (https://github.com/patrickemonts/zettelkasten/pull/4/commits/1b86664efdf92d7ce8e39e3893197193b3a74353), the note is not fully written yet. I will add it later.

File: vault/Ground State Problem.md
Comment (patrickemonts) [reply vault/Ground State Problem.md:1]: Added in https://github.com/patrickemonts/zettelkasten/pull/4/commits/cbec2bbfe323892c9d8f866f42f78ca4c0815551, also added a node about the time-independent Schroedinger equation

File: vault/Variational Quantum Algorithm (VQA).md
Comment (patrickemonts) [reply vault/Variational Quantum Algorithm (VQA).md:1]: Done, we can add further references later.

**Review by AdrianPerezSalinas — APPROVED** [review]

**Pull request merged**